patches/minecraft/net/minecraft/server/packs/resources/FallbackResourceManager.java.patch

```diff
@@ -1,12 +0,0 @@
---- a/net/minecraft/server/packs/resources/FallbackResourceManager.java
-+++ b/net/minecraft/server/packs/resources/FallbackResourceManager.java
-@@ -53,6 +_,9 @@
- 
-    private void m_215395_(String p_215396_, @Nullable PackResources p_215397_, @Nullable Predicate<ResourceLocation> p_215398_) {
-       this.f_10599_.add(new FallbackResourceManager.PackEntry(p_215396_, p_215397_, p_215398_));
-+      if (p_215397_ != null) {
-+         p_215397_.initForNamespace(this.f_10602_);
-+      }
```

Comment:
See PR message, it was what was causing the performance degradation.
It is not needed, since the packs are already initialized when they get to this point, causing double initialization.